GameProject/Assets/OpenWorld Assets/Scripts/trdprsnCharacter.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class trdprsnCharacter : MonoBehaviour
+{
+
+    [SerializeField] float movespeed;
+    [SerializeField] float rotatespeed;
+    //private float rotX;
+    //private float rotY;
+    [SerializeField] float rotZ;
```

Comment:
all of the above private fields do not meet the convention of the codebase. Also missing access modifiers.